entc/gen/predicate.go

```diff
@@ -67,6 +67,6 @@ var (
 	boolOps     = []Op{EQ, NEQ}
 	enumOps     = append(boolOps, In, NotIn)
 	numericOps  = append(enumOps, GT, GTE, LT, LTE)
-	stringOps   = append(numericOps, Contains, HasPrefix, HasSuffix)
+	stringOps   = append(numericOps, EqualFold, Contains, ContainsFold, HasPrefix, HasSuffix)
```

Comment:
The codegen will fail in case the storage-driver doesn't support EqualFold and ContainsFold. That's why the `Ops` appends the driver predicates.